app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -281,107 +283,111 @@ public void onClick(View view) {
 
     @Override
     public void onItemClick(View childView, final int position) {
-        if (!checkNetwork(this, parent)) return;
-
-        positionShareOption = position;
-        if (sharableAccountsList.get(position) == OTHERS) {
-            triedUploading = true;
-            shareToOthers();
-            return;
-        }
+        ConnectivityManager cm = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo ni = cm.getActiveNetworkInfo();
+        if (ni == null) {
+            Snackbar.make(findViewById(android.R.id.content), getString(R.string.internet_is_off), Snackbar.LENGTH_SHORT).show();
```

Comment:
This snackbar should be replaced with a toast giving an appropriate message with LENGTH_LONG.